### DISCUSSION TITLE
UnorderedList list-style css discrepancy compared to Evergreen

### DISCUSSION THREAD

**Discussion by wlumtwilio** [twilio-labs/paste#3318] [General] [answered]
Hi team,

Having a reset style will override the list-style of the UnorderedList Paste component. But if I use the Evergreen component, it does not. I have a css file that overrides the list-style and would like to display bullet points using Paste, the same way Evergreen was behaving.

Example with Paste vs Evergreen: https://codesandbox.io/s/fervent-monad-djstcf?file=/src/index.tsx

This is the same issue as https://github.com/twilio-labs/paste/issues/2954 which is marked closed. However in the Sandbox, the issue still remains.

**Comment by SiTaggart** [comment]
Hi @wlumtwilio 

Well, that's embarrassing. Seems like that fix never actually worked 😰

I've put up a new PR that should actually fix it now. You can follow along here https://github.com/twilio-labs/paste/pull/3319.

We'll need to do a publish sometime later this week before it's out on NPM.

**Reply by SiTaggart** [reply, marked as answer]
This is now fixed and on NPM.

**Reply by wlumtwilio** [reply]
Thank you for the fast fix! Confirmed that it's working as intended
